environment.yml

```diff
@@ -2,7 +2,7 @@ name: SNAPRed
 channels:
 - conda-forge
 - default
-- mantid
+- mantid/label/nightly
```

Comment:
This part makes me uncomfortable

Reply:
Its fine, the plan going forward is to pin versions in qa and main with every release.  Its a bit of a compromise as we cant wait for the mantid release cycle to move forward with these changes.